gui/src/TCWin.cpp

```diff
@@ -411,13 +411,27 @@ void TCWin::RecalculateSize() {
   }
 }
 
+/**
+ * Handle OK button click - close dialog and perform cleanup.
+ */
 void TCWin::OKEvent(wxCommandEvent &event) {
   Hide();
-  pParent->pCwin = NULL;
+
```

Comment:
Just a driive by comment: these functions should be documented in the header file, not the cpp one.

Reply:
I've fixed the code comments.